app/components/header/header-navigation.njk

```diff
@@ -25,7 +22,8 @@
         },
         {
           'url'  : 'https://www.nhs.uk/conditions/social-care-and-support/',
-          'label' : 'Care and support'
+          'label' : 'Care and support',
+          'active': true
```

Comment:
Any thinking on the name for this? `active` vs `current`? I might have expected current because that's the wording used on the aria and class?

Reply:
I copied the param names from the new(ish) [GOVUK Service navigation](https://design-system.service.gov.uk/components/service-navigation/) component.

There they use both `active` and `current`, with subtly different semantics - `current` meaning that it’s the link to the exact same page you're already on, and `active` meaning it's a link to the parent page for the current section you’re in.

Both are visually the same though, and the only different is the aria attribute - so I suspect it’ll often not get implemented 100% correctly?

In my head something like `currentPage: true` and `currentSection: true` might be more intuitive, but I guess it’s a trade-off between alignment with govuk vs doing something that might be better?